### PR TITLE
chore(tooling): update build pipeline to npm 5

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

**Comment by ianwremmel** [comment]
maybe. it might actually be enough to just let jenkins run `npm install` and commit `package-lock.json` at that point since we're not directly merging PRs.

**Comment by ianwremmel** [comment]
but, that's also part of why i haven't `package-lock` yet :)

**Comment by ianwremmel** [comment]
blocked by https://github.com/npm/npm/issues/16807

File: docker/builder/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,8 @@ RUN tar -xzf "sc-${SAUCE_CONNECT_VERSION}-linux.tar.gz"
 RUN ln -s "${SC_BINARY}" /usr/local/bin/sc
 RUN chmod ugo+w /sauce
 
+RUN npm install -g npm@latest
```
Comment (adamweeks) [inline docker/builder/Dockerfile]: Should we do `npm@5` instead?

Comment (ianwremmel) [inline docker/builder/Dockerfile]: maybe, but it doesn't much matter right now. until https://github.com/npm/npm/issues/16807 gets fixed, we can't use npm 5 in the pipeline.

**Comment by ianwremmel** [comment]
closing for now.

**Pull request closed without merging**